pkgs/development/libraries/xtensor/default.nix

```diff
@@ -2,36 +2,49 @@
 , stdenv
 , fetchFromGitHub
 , cmake
-, gtest
+, doctest
+, useXsimd ? true # Test with xsimd on hydra
+, enableAssertions ? true
+, enableBoundChecks ? true # Verify that the test-suite passes bound checks
```

Comment:
Motivated by https://github.com/xtensor-stack/xtensor/issues/2685

Reply:
As https://github.com/xtensor-stack/xtensor/issues/2685#issuecomment-1511922726 points out, `enableBoundChecks` are expected to break the test-suite ✅